src/jobs/jobs.service.ts

```
@@ -1,387 +1,384 @@
+
+//     return query.getMany();
+//   }
+
+//   async findJobById(id: number, includeDeleted: boolean = false): Promise<Job> {
+//     const query = this.jobRepository.createQueryBuilder('job')
+//       .where('job.id = :id', { id });
+
+//     if (!includeDeleted) {
+//       query.andWhere('job.deletedAt IS NULL');
+//     }
     
```

ESLint

[error] Delete `····`

(prettier/prettier)